LICENSE-MIT

```diff
@@ -1,3 +1,5 @@
+Add header to MIT License
```

Comment:
This looks like a note about making change. Did you mean to write the actual change here?

Reply:
Went a bit fast :D Fixed now